onnxmltools/convert/sklearn/operator_converters/NaiveBayes.py

```diff
@@ -0,0 +1,100 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+from ....proto import onnx_proto
+from ...common._registration import register_converter
+
+
+def convert_sklearn_naive_bayes(scope, operator, container):
+    # Multinomial NB
```

Comment:
Could you also draw the computational graph with all shape specified? I am not sure if you need those Reshape's below and would like to have more information about your conversion. You're using the shape calculator for linear classifier (which produces two outputs) the first equation below only shows one output.